src/bench_internal.c

```diff
@@ -17,6 +17,31 @@
 #include "ecmult_impl.h"
 #include "bench.h"
 
+static void help(int default_iters) {
+    printf("Benchmarks internal routines for the following areas:\n");
+    printf("    - Scalar operations (modulo the curve's order)\n");
+    printf("    - Field operations (modulo the curve's field size)\n");
+    printf("    - Group operations (both in Jacobian and affine coordinates)\n");
+    printf("    - Point multiplication\n");
+    printf("    - Hash algorithms\n");
+    printf("    - Context object handling\n");
```

Comment:
Perhaps it suffices to say "Benchmarks various internal routines". This list is mostly redundant to the list below. If we omit it, we won't need to (remember to) touch two lists every time we add or remove operations.

Reply:
Agree, done.